pipeline/Makefile

```diff
@@ -119,6 +124,7 @@ run-task: ## Running a specific task
 PIPELINE_ARGS := --PipelineParams-output-dir /files/data/output --PipelineParams-file-parent-dir /files/data --PipelineParams-date $(DATA_DATE) --PipelineParams-gene-config-path $(GENE_CONFIG_FILE)
 
 force-run-task: ## Running a specific task (deleting its dependencies first)
+	## TODO: should this command use RUN_ARGS command twice?
```

Comment:
interestingly, removing one changes the output and throws a new error. i'm going to leave as is with this todo since it's irrelevant to the changes in this PR